contracts/guards/DelegateCallTransactionGuard.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity >=0.7.0 <0.9.0;
+
+import "../base/GuardManager.sol";
+import "../GnosisSafe.sol";
+
+contract DelegateCallTransactionGuard is Guard {
+
+    address immutable public allowedTarget;
+
+    constructor(address target) {
+        allowedTarget = target;
+    }
+
+    fallback() external {
+        // We only check known calls
+    }
+
+    function dataSeletor(bytes calldata _bytes)
```

Comment:
`dataSeletor` -> `dataSelector`

Reply:
you don't know the work `seletor`? .... google does neither so I will rename it :P